docs/getting-started-guide/index.rst

```diff
@@ -4,6 +4,29 @@ Getting Started
 
 The getting started guide aims to get you using xCDAT productively as quickly as possible.
 
+.. grid:: 1 1 2 2
+    :gutter: 2
+
+    .. grid-item-card:: Project Overview
+        :img-top: ../_static/icons/dashboard.png
```

Comment:
I downloaded png and then noticed you used svg. I can change this (the file sizes are small either way).

Reply:
Thanks for adding these links. `.png` is fine